### PR TITLE
Aligned "drag handles..." to center

### PR DESCRIPTION
**Pull request by ashwinpn**
The text which describes the time window slider is not aligned properly (Namely, "Drag handles below to change time window") - aligning to the center makes it look better.

### CODE REVIEW

File: make_figures.py [hunk 1]
```diff
@@ -195,7 +195,8 @@ def make_timeplot(df_measure, df_prediction):
             showarrow=False,
             font_size=LABEL_FONT_SIZE - 6,
             font_color="DarkSlateGray",
-            text="Drag handles below to change time window")
+            text="Drag handles below to change time window",
+            align="center")
```
Comment (GaelVaroquaux) [inline make_figures.py]: Definitely seems like an improvement. Shouldn't we right-align and put x=1? That way it's more likely to be on top of the handles.

**Comment by GaelVaroquaux** [comment]
Definitely seems like an improvement. Shouldn't we right-align and put x=1? That way it's more likely to be on top of the handles.

File: make_figures.py (hunk 1 above)
Comment (ashwinpn) [reply make_figures.py]: I see what you mean.

**Comment by GaelVaroquaux** [comment]
Very cool! Thanks!

**Pull request merged**